src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -909,5 +909,61 @@ namespace wallet_rpc
     };
   };
 
+  struct COMMAND_RPC_GET_LANGUAGES
```

Comment:
Discussed this with @moneromooo-monero a week ago and decided it didn't need it.